contrib/tracking.h

```diff
@@ -0,0 +1,70 @@
+
+#ifndef _OPENCV3_MY_H_
```

Comment:
Why is this `MY` here and not something `TRACKING`?

Reply:
k,k.